### ISSUE TITLE
local validator doesn't gracefully shut down on app close

### ISSUE THREAD

**Issue by SvenDowideit** [workbenchapp/solana-workbench#287]

**Comment by sammanadh** [comment]
hey @SvenDowideit, by "Stop&Delete" you mean remove the container or delete the entire image?

**Comment by sammanadh** [comment]
ohhhh. it must mean remove the container right. cause there one button for that in validator page.

**Comment by SvenDowideit** [comment]
yup, you got it :) - its actually detecting the close event, preventing it, and asking questions that looked to be challenging when I had a quick look :)
